src/test/java/io/supertokens/test/session/CleanExpiredAccessTokenSigningKeysTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *    Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+package io.supertokens.test.session;
+
+import io.supertokens.ProcessState;
+import io.supertokens.config.Config;
+import io.supertokens.cronjobs.cleanExpiredAccessTokenSigningKeys.CleanExpiredAccessTokenSigningKeys;
+import io.supertokens.test.TestingProcessManager;
+import io.supertokens.test.Utils;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+
+public class CleanExpiredAccessTokenSigningKeysTest {
```

Comment:
You need to add a test in which there are a few keys in the db, some of which are expired, the cronjob runs, and then removes just the expired keys.